src/ya-comiste-meta/schema.graphql

```diff
@@ -26,6 +46,8 @@ type SDUICardComponent {
 
 type Query {
   mobileEntrypointSections(key: String!): [SDUISection!]!
+  "Returns information about the current user (can be authenticated or anonymous)."
+  whoami: WhoamiPayload!
```

Comment:
Why not use a union type?

Reply:
Because I find it unnecessarily complicated. It's there just for debugging purposes and it doesn't have any other real usage which would require better structure.